pkg/api/v1alpha1/cluster_webhook_test.go

```diff
@@ -833,6 +833,24 @@ func TestClusterValidateUpdateAWSIamNameImmutableAddConfig(t *testing.T) {
 	g.Expect(c.ValidateUpdate(cOld)).NotTo(Succeed())
 }
 
+func TestClusterValidateUpdateUnsetBundlesRefImmutable(t *testing.T) {
+	cOld := createCluster()
+	c := cOld.DeepCopy()
+	c.Spec.BundlesRef = nil
+
+	g := NewWithT(t)
+	g.Expect(c.ValidateUpdate(cOld)).NotTo(Succeed())
+}
+
+func TestClusterValidateUpdateBundleRefRollbackInvalid(t *testing.T) {
```

Comment:
Could we add a test for:
* bundle moving up
* bundle staying the same
* updating bundle to a custom name

Reply:
Will add test for bundle moving up and updating bundle to custom name. Bundle staying the same should be covered by other tests.